requirements.txt

```diff
@@ -12,5 +12,4 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-th2-grpc-common~=3.1.2
-grpcio-tools==1.33.1
```

Comment:
Don't we need this during generation?

Reply:
`grpcio-tools` is the dependency of `th2-grpc-common`, so it will be included transitive.